src/Form.php

```diff
@@ -502,7 +513,7 @@ public function loadPOST()
     {
         $post = $_POST;
 
-        $this->hook('loadPOST', [&$post]);
+        $this->hook(self::HOOK_LOAD_POST, [&$post]);
```

Comment:
What if I extend this class? Shouldn't use `static` instead in this case and similar cases elsewhere?